Detectors/FIT/T0/macros/readHitsDigits.C

```diff
@@ -0,0 +1,87 @@
+#if !defined(__CLING__) || defined(__ROOTCLING__)
+#include "FITSimulation/Detector.h"
+#include "FITBase/Digit.h"
+#include <TH2F.h>
+#endif
+
+void readHitsDigits()
+{
+  using namespace o2::fit;
```

Comment:
do you nee o2::fit namespace declaration here? Below you use explicitly ``o2::fit::Digit``.